src/test/java/com/easypost/BatchTest.java

```diff
@@ -206,22 +209,23 @@ public void testAddRemoveShipment() throws EasyPostException {
      */
     @Test
     public void testLabel() throws EasyPostException, InterruptedException {
-        _vcr.setUpTest("label");
+        vcr.setUpTest("label");
 
         Batch batch = createOneCallBuyBatch();
 
         batch = batch.buy();
 
-        if (_vcr.isRecording()) {
-            Thread.sleep(5000); // Wait enough time for processing
+        if (vcr.isRecording()) {
+            Thread.sleep(10000); // Wait enough time for processing
```

Comment:
5 seconds wasn't enough the first time, was the second time. Bumped it to 10 seconds to give it plenty of buffer.

Reply:
Sounds good. I wonder if it was a fluke or that these compiled languages take a bit longer to process these requests as the dynamic ones seem to always finish under 5. Either way, this is fine.